### PR TITLE
Topic/solgs/refactor jan23

### PR DESCRIPTION
**Pull request by isaak**
Description 
-----------------------------------------------------

-- reorganize, rename, clean up solgs mason components

Checklist 
---------------------------------------------------------------------------------------
- [ ] Refactoring only
- [ ] Documentation only
- [ ] Fixture update only
- [ ] Bug fix
  - [ ] The relevant issue has been closed.
  - [ ] Further work is required.
- [ ] New feature
  - [ ] Relevant tests have been created and run.
  - [ ] Data was added to the fixture
    - [ ] Data was added via a patch in `/t/data/fixture/patches/`.
  - [ ] User-Facing Change
    - [ ] The user manual in `/docs` has been updated.
  - [ ] Any new Perl has been documented using **perldoc**.
  - [ ] Any new JavaScript has been documented using **JSDoc**.
  - [ ] Any new _legacy_ JavaScript has been moved from `/js` to `/js/source/legacy`.

### CODE REVIEW

File: mason/solgs/tools/cluster/index.mas [hunk 1]
```diff
@@ -41,7 +41,7 @@ my $url = $c->req->path;
 
 <!-- % if ($url =~ /cluster\/analysis/) { -->
 
-<!-- <& /solgs/genotyping_protocol.mas &> -->
+<!-- <& /solgs/genotyping_protocol/inde.mas &> -->
```
Comment (alockrow) [inline mason/solgs/tools/cluster/index.mas:44]: This looks like a typo!
```suggestion
<!-- <& /solgs/genotyping_protocol/index.mas &> -->
```

**Review by alockrow — CHANGES_REQUESTED** [review]
Found one typo, otherwise looks good!

**Review by alockrow — APPROVED** [review]
:-)

**Review by lukasmueller — APPROVED** [review]
:-)

**Pull request merged**